src/script/auth/page/TeamName.tsx

```diff
@@ -64,37 +64,32 @@ interface DispatchProps {
   resetInviteErrors: () => Promise<void>;
 }
 
-interface State {
-  enteredTeamName: string;
-  error: Error;
-  isValidTeamName: boolean;
-}
-
-class TeamName extends React.Component<Props & ConnectedProps & DispatchProps & InjectedIntlProps, State> {
-  private readonly teamNameInput: React.RefObject<any> = React.createRef();
-  state: State = {
-    enteredTeamName: this.props.teamName || '',
-    error: null,
-    isValidTeamName: !!this.props.teamName,
-  };
+const TeamName = ({intl: {formatMessage: _}, ...props}: Props & ConnectedProps & DispatchProps & InjectedIntlProps) => {
```

Comment:
Let's destructure the props here

Reply:
Most of the time they are used just once and i would argue that it is easier to read and understand where the values are coming from if we address them by `props.value`.

Reply:
I would argue it's more readable that way because the actual code is more concise. Also that's what we do in new code.